src/main/java/net/clesperanto/imagej/ImageJDataType.java

```diff
@@ -9,18 +9,20 @@
 
 // TODO add all types ImagePlus.GRAY8, ImagePlus.GRAY16, ImagePlus.GRAY32, ImagePlus.COLOR_256 or ImagePlus.COLOR_RGB)
 public enum ImageJDataType {
-    FLOAT32(DataType.fromString("float"), ImagePlus.GRAY32, float[].class),
-    UINT16(DataType.fromString("ushort"), ImagePlus.GRAY16, short[].class),
-    UINT8(DataType.fromString("uchar"), ImagePlus.GRAY8, byte[].class);
+    FLOAT32(DataType.fromString("float"), ImagePlus.GRAY32, float[].class, 32),
+    UINT16(DataType.fromString("ushort"), ImagePlus.GRAY16, short[].class, 16),
+    UINT8(DataType.fromString("uchar"), ImagePlus.GRAY8, byte[].class, 8);
```

Comment:
```suggestion
    FLOAT32(DataType.fromString("float"), ImagePlus.GRAY32, float[].class, blip),
    UINT16(DataType.fromString("ushort"), ImagePlus.GRAY16, short[].class, blop),
    UINT8(DataType.fromString("uchar"), ImagePlus.GRAY8, byte[].class, blap);
```

this is a crappy test